ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/capi/provision/cucushift-installer-rehearse-aws-ipi-capi-provision-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-capi-provision 
+  steps:
+  - chain: ipi-conf-aws
+  - chain: ipi-install
+  - ref: capi-conf-apply-feature-gate        
+  documentation: |-
+    Create an IPI cluster with featuregate enabled (capi) on AWS for QE e2e tests starting v-4.11
```

Comment:
How about:
s/ starting v-4.11/. This is starting from 4.11/

Reply:
Done